auto_submit/lib/service/revert_review_template.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class RevertReviewTemplate {
+  RevertReviewTemplate({
+      required this.repositorySlug,
+      required this.revertPrNumber, 
+      required this.revertPrAuthor,
+      required this.originalPrLink,}) {
+    constructTitle();
+    constructBody();
+  }
+
+  String repositorySlug;
```

Comment:
These should all be final since they're injecting in the constructor

Reply:
Done.